README.md

```diff
@@ -1,13 +1,15 @@
 # containerlab-vms
 
-This project is meant to help building containerlab compatible containers, running virtual routers inside of them.  This has been highly inspired by the vrntelab project.  The working principle is the same as the aforementioned project, but the inner structure of the code has been revised to fix stability issues we have encountered.
+This project is meant to help building [containerlab](https://containerlab.srlinux.dev/) compatible containers, running virtual routers inside of them.  This has been highly inspired by the vrntelab project.  The working principle is the same as the aforementioned project, but the inner structure of the code has been revised to fix stability issues we have encountered.
 
 ## Supported vendors
 
  1. `Cisco`
     1. [`xrv`](vms/xrv)
     2. [`xrv9k`](vms/xrv9k)
 
+You can find some examples of labs using the above vendors in [/examples](examples/).
+
 ## Package structure
```

Comment:
I would add a short getting started section here, with step by step instructions. Now it is kind of difficult so see all the steps. 

I think it will be something like: 

1. get a supported router image from the vendor (we can't distribute these for copyright/licensing issues)
2. put it in the folder ...
3. build base container: ...
4. build router container: ...
5. start the lab: ...

Reply:
Also, the package structure section, I would move it to the back or remove it, as it provides no explanation or comment: it is just a blank directory listing.

Reply:
Done in https://github.com/inmanta/containerlab-vms/pull/5/commits/cfe2a7a0e207d29c9a33e6235c0373562c22ee8f